po/tr.po

```diff
@@ -631,7 +633,7 @@ msgstr "oynatıldı"
 
 #: src/download.cpp:83
 msgid "unknown (bug)."
-msgstr "bilinmiyor (bug) "
+msgstr "bilinmiyor (hata) "
```

Comment:
This has a space at the end, where a period should be.

Reply:
Fixed.